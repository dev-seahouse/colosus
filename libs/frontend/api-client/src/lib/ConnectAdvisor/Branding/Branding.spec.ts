import ConnectAdvisorBrandingApi from './Branding';

describe('Branding', () => {
  const connectAdvisorBrandingApi = new ConnectAdvisorBrandingApi();

  describe('getTenantBranding()', () => {
    it('should return a valid response', async () => {
      const res = await connectAdvisorBrandingApi.getTenantBranding();

      expect(res.data).toMatchSnapshot();
    });
  });
});
