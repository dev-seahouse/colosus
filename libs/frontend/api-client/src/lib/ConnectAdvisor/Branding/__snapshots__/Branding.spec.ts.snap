// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Branding > getTenantBranding() > should return a valid response 1`] = `
{
  "brandColor": "#00876A",
  "headerBgColor": "#fff",
  "logoUrl": null,
  "tradeName": "Wealth Avenue",
}
`;
